app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -62,22 +63,23 @@
 
         <com.google.android.material.textfield.TextInputEditText
             android:id="@+id/input_name"
-            android:fontFamily="@font/roboto_light_italic"
             android:layout_width="match_parent"
-            android:layout_height="50dp" />
+            android:layout_height="@dimen/signUp_50dp"
```

Comment:
remember not to tie dimens to specific screens when they can be reusable

Reply:
i'll implement it, thanks